site/layouts/partials/head-meta.html

```diff
@@ -0,0 +1,4 @@
+
+<meta name="description" content="{{ . }}">
+<meta name="twitter:description" content="{{ . }}">
+<meta name="og:description" content="{{ . }}">
```

Comment:
Should we also add `og:title` and `og:site_name`?

Reply:
I think it makes sense, but that's a question for SEO/Content that I can't answer.

Reply:
I have updated the JIRA ticket asking the SEO team if we should include those and what values to use.

Reply:
Update include `og:title` and `og:site_name` queued for next commit.